NEWS

```diff
@@ -12,6 +12,10 @@
   which have empty repository scopes (gh#SUSE/machinery#1268)
 * Inspection of unmanaged-files is now also using the faster machinery-helper
   when the files are extracted
+* Introduce format version 5 which adds the "environment" scope,
+  "environment" being a hidden scope only used internaly has no impact on users
```

Comment:
Maybe better two sentences? "Introduce format version 5 which adds the "environment" scope. This scope is a hidden scope which is used internally by Machinery."

"has no impact on users" is not completely true, because it requires the format upgrade. That's why we are mentioning it here.